src/server/scripts/Commands/cs_gobject.cpp

```diff
@@ -416,25 +416,37 @@ class gobject_commandscript : public CommandScript
         }
 
         char* orientation = strtok(NULL, " ");
-        float o;
+        float oz, oy, ox;
 
         if (orientation)
-            o = (float)atof(orientation);
+            oz = (float)atof(orientation);
         else
         {
             Player* player = handler->GetSession()->GetPlayer();
-            o = player->GetOrientation();
+            oz = player->GetOrientation();
         }
 
-        object->Relocate(object->GetPositionX(), object->GetPositionY(), object->GetPositionZ(), o);
-        object->UpdateRotationFields();
+        // try extract two opt-parameters
+        orientation = strtok(NULL, " ");
+        if (orientation)
```

Comment:
https://travis-ci.org/TrinityCore/TrinityCore/builds/50880616#L1045
>variable 'ox' is used uninitialized whenever 'if' condition is false

was this part tested ingame (or even debugged) ?